core/trino-main/src/main/java/io/trino/cost/TableScanStatsRule.java

```diff
@@ -105,4 +107,23 @@ else if (type instanceof FixedWidthType) {
         });
         return result.build();
     }
+
+    private static double getNullsFraction(ColumnStatistics columnStatistics, Estimate rowCount)
+    {
+        if (columnStatistics.getNullsFraction().isUnknown() && !columnStatistics.getDistinctValuesCount().isUnknown()) {
+            // When NDV is greater than or equal to row count, there are no nulls
+            if (Double.compare(columnStatistics.getDistinctValuesCount().getValue(), rowCount.getValue()) >= 0) {
```

Comment:
why not just `columnStatistics.getDistinctValuesCount().getValue() >= rowCount.getValue()`